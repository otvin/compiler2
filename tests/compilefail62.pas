program compilefail62;
{try to write without defining 'output' as a program parameter}

begin
	writeln(0);
end.
