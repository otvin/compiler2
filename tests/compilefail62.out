writeln() called without 'output' defined in file:tests/compilefail62.pas line:5     column:5  
